package.json

```diff
@@ -41,6 +41,7 @@
     "lint": "eslint \"packages/**/*.ts\" \"**/__tests__/**/*.ts\"",
     "lint:packages": "eslint \"packages/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
     "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
+    "circularDependencyCheck": "madge -c lib",
```

Comment:
I know this is probably already part of the gulp tasks, but curious if we will ever move things to just npm scripts and include this as part of the build 😋

Reply:
I hope things get moved eventually. It'd be nice to be able to remove gulp.

Reply:
Based on internal discussion, seems like we have sequential tasks using Gulp that don't need to be, and their performance could be improved by degulpifying them 🙂 

For more complex tasks like parallel commands (watch = tsc + lint) or web help bundle generation, it's probably fine to keep using Gulp for now.

Reply:
Created #686 to track this